packages/experimental/Avatar/src/Avatar.types.ts

```diff
@@ -0,0 +1,101 @@
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { ImageProps, ViewProps, ImageURISource, TextProps, ColorValue } from 'react-native';
+import { IBackgroundColorTokens, IForegroundColorTokens } from '@fluentui-react-native/tokens';
+
+export const avatarName = 'Avatar';
+
+export type AvatarSize = 'size8' | 'size24' | 'size32' | 'size40' | 'size48' | 'size56' | 'size72' | 'size100' | 'size120';
+
+/**
+ * Sets color of the avatar when there is no picture. Uses fluent color names
+ */
+export type AvatarColor =
+  | 'cornflower'
+  | 'blue'
+  | 'royalBlue'
+  | 'teal'
+  | 'forest'
+  | 'darkGreen'
+  | 'berry'
+  | 'hotPink'
+  | 'grape'
+  | 'purple'
+  | 'pumpkin'
+  | 'red'
+  | 'burgundy'
+  | 'orchid'
+  | 'brass'
+  | 'darkRed'
+  | 'beige'
+  | 'platinum'
+  | 'steel'
+  | 'brown';
+
+export type AvatarPresence = 'none' | 'offline' | 'online' | 'away' | 'dnd' | 'blocked' | 'busy';
+
+export type RingThickness = number | 'xSmall' | 'small' | 'medium' | 'large' | 'xlarge' | 'xxlarge';
+
+export interface RingConfig {
+  accent?: boolean;
+  transparent?: boolean;
+  ringColor?: ColorValue; // glow
+  ringBackgroundColor?: ColorValue; // inner/outer rings
+  ringThickness?: RingThickness;
+  innerGap?: RingThickness;
+}
+
+export interface AvatarConfigurableProps {
+  size?: AvatarSize;
+  ring?: RingConfig;
+  coinColorFluent?: AvatarColor;
+}
+
+export interface AvatarProps extends IViewProps, AvatarConfigurableProps {
```

Comment:
It seems like we would want to expose the AvatarTokens we have as props as well, so that they can be passed directly to the component